File: PHP/BinarySearch.php
<?php
/* Binary Search implementation in PHP */
function binarySearch(Array $arr, $item) {
    $low = 0;
    $high = count($arr) - 1;
	$found = false;
	while ($low <= $high) {
		$mid = $low + ($high - $low) / 2; //Midpoint of the Search Space
		if ($arr[$mid] === $item) { //Found the element
			$found = true;
			break;
		} else {
			if ($item < $arr[$mid]) { 
				$high = $mid - 1;
			} else { 
				$low = $mid + 1;
			}
		}
	}
	return $found;
}
?>

File: Python/BinarySearch.py
def binarySearch(arr,n,a):
    top=0
    bot=n-1
    while top<=bot:
        mid=int((top+bot)/2);
        if  a==arr[mid]:
            return mid
        elif a<arr[mid]:
            bot=mid-1
        elif a>arr[mid]:
            top=mid+1
    return -1

